skyportal/tests/api/test_annotation_services.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from skyportal.tests import api
+
+
+@pytest.mark.flaky(reruns=2)
+def test_irsa_wise(annotation_token, public_source, public_group):
+    status, data = api(
+        'POST',
+        f'sources/{public_source.id}/annotations/irsa',
+        token=annotation_token,
+        data={'crossmatchRadius': 30},
+    )
+    assert status == 200
+
+    status, data = api(
+        'GET',
+        f'sources/{public_source.id}/annotations',
+        token=annotation_token,
+    )
+
+    assert status == 200
+
+    assert any(
```

Comment:
Why is this `any` and not `all`. Also are you sure the public source coordinates are never going to change?